llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -300,14 +305,14 @@ let FPDPRounding = 1, isReMaterializable = 0 in {
   let OtherPredicates = [UseRealTrue16Insts] in
     defm V_CVT_F16_F32_t16 : VOP1Inst <"v_cvt_f16_f32_t16", VOPProfile_True16<VOP_F16_F32>, any_fpround>;
   let OtherPredicates = [UseFakeTrue16Insts] in
-    defm V_CVT_F16_F32_fake16 : VOP1Inst <"v_cvt_f16_f32_fake16", VOPProfile_Fake16<VOP_F16_F32>, any_fpround>;
+    defm V_CVT_F16_F32_fake16 : VOP1Inst<"v_cvt_f16_f32_fake16", VOP_F16_F32_Fake16_OP_SEL, any_fpround>;
```

Comment:
Can it use VOP1Inst_t16 instead? It will define both _t16 and _fake16.

Reply:
Probably not, since we only want to use `VOP_F16_F32_Fake16_OP_SEL` for `_fake16`.

Reply:
The comment above this might explain the reason:

```
  // V_CVT_F16_F32 and V_CVT_F32_F16 are special cases because they are
  // present in targets without Has16BitInsts. Otherwise they could use
  // class VOP1Inst_t16
```